projects/goharbor/harbor/Makefile

```diff
@@ -0,0 +1,113 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
```

Comment:
To fix gather lionesses/attribution, this project pulls in some json libs from xeipuuv which is not setup exactly how go-licenses expects.  We have handled this in the flux controllers since they pull in the same deps.  If you add something like this to this makefile, it should fix it up.  I verified this locally.

https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/fluxcd/source-controller/Makefile#L33